flow-decomposition/src/main/java/com/powsybl/flow_decomposition/rescaler/DecomposedFlowRescalerNoOp.java

```diff
@@ -8,20 +8,26 @@
 package com.powsybl.flow_decomposition.rescaler;
 
 import com.powsybl.flow_decomposition.DecomposedFlow;
+import com.powsybl.iidm.network.Network;
 
 /**
  * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
  * @author Hugo Schindler {@literal <hugo.schindler at rte-france.com>}
  * @author Caio Luke {@literal <caio.luke at artelys.com>}
  */
-public class DecomposedFlowRescalerNoOp implements DecomposedFlowRescaler {
+public class DecomposedFlowRescalerNoOp extends AbstractDecomposedRescaler {
 
     public DecomposedFlowRescalerNoOp() {
         // empty constructor
     }
 
     @Override
-    public DecomposedFlow rescale(DecomposedFlow decomposedFlow) {
-        return decomposedFlow;
+    protected boolean shouldRescaleFlows(DecomposedFlow decomposedFlow) {
+        return false;
+    }
+
+    @Override
+    protected RescaledFlows computeRescaledFlows(DecomposedFlow decomposedFlow, Network network) {
+        return null;
```

Comment:
Maybe it should raise an exception has by design we cannot rescale with such rescaler